Controller/AudioPlayer.cs

```diff
@@ -114,6 +114,7 @@ public virtual void Prev()
          */
         public void PlaySong(Song song)
         {
+            WaveOutDevice.Stop();

```

Comment:
hier moet een if statement omheen anders gaat het stuk

Reply:
Waarom gebeurd dit überhaupt?

Reply:
Als ik in het search screen een nieuw nummer aan klik terwijl die een nummer afspeel krijg ik een re-initialize error

Reply:
That's why

Reply:
maar, de oplossing lijkt me toch niet om maar gewoon altijd het nummer te stoppen?
Behoud hij dan gewoon z'n voortgang als je pause, play doet?

Reply:
ik zal zo wel even overleggen met vincent

Reply:
ja paul goeie vraag/ hoe werkt dat?

Reply:
heb er nu een if tussen gezet
maar dit zou alleen moeten gebeuren als die nog speelt